plugins/powerd/fu-plugin-powerd.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2021 Twain Byrnes <binaryNewts@google.com>
+ *                    George Popoola <gpopoola@google.com>
```

Comment:
You need to follow the existing style, e.g. this should be

     * Copyright (C) 2021 Twain Byrnes <binaryNewts@google.com>
     * Copyright (C) 2021 George Popoola <gpopoola@google.com>